src/calendar/day/basic/index.js

```diff
@@ -92,26 +93,29 @@ class Day extends Component {
     }
 
     return (
-      <TouchableOpacity
-        testID={this.props.testID}
-        style={containerStyle}
-        onPress={this.onDayPress}
-        onLongPress={this.onDayLongPress}
-        activeOpacity={activeOpacity}
-        disabled={shouldDisableTouchEvent}
-        accessibilityRole={isDisabled ? undefined : 'button'}
-        accessibilityLabel={this.props.accessibilityLabel}
-      >
-        <Text allowFontScaling={false} style={textStyle}>{String(this.props.children)}</Text>
-        <Dot
-          theme={theme}
-          isMarked={marked}
-          dotColor={dotColor}
-          isSelected={selected}
-          isToday={isToday}
-          isDisabled={isDisabled}
-        />
-      </TouchableOpacity>
+      <View>
```

Comment:
Do we need a `View` here? Could we use a fragment so we would not have to create an additional View in the native hierarchy?

Reply:
Here have add view to warp the DayView and DayUnderView.